packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -254,6 +272,14 @@ function NoteEditor(props: Props, ref: any) {
 			redo: function() {
 				webviewRef.current.injectJavaScript('cm.redo(); true;');
 			},
+			select: (anchor: number, head: number) => {
+				const anchorStr = JSON.stringify(anchor),
+					headStr = JSON.stringify(head);
```

Comment:
Just `const headStr = JSON.stringify(head);` (we avoid defining multiple variables in one statement)

Reply:
Or even better don't create an intermediate variable and escape the data directly in the string like you've done below.